src/ingest/generation.rs

```diff
@@ -0,0 +1,45 @@
+use std::sync::atomic::{AtomicUsize, Ordering};
+use std::sync::Arc;
+
+use chrono::{Datelike, Utc};
+use tokio::sync::RwLock;
+
+pub struct SequenceGenerator {
+    counter: AtomicUsize,
+    last_reset_date: RwLock<u32>,
```

Comment:
I believe we can achieve better performance with `AtomicU32` because it eliminates the need for locking mechanisms, which can cause contention when multiple threads are involved.

While the volume of op logs may not be particularly high, Giganto is a performance-critical module. Therefore, even if the overhead of lock might seem minor given the current op log frequency, I think it's worth optimizing.

Reply:
Changed to `AtomicU32` has been completed.
Thanks you.